ddtrace/tracer/spancontext_test.go

```diff
@@ -40,6 +42,51 @@ func TestNewSpanContextPushError(t *testing.T) {
 	}
 }
 
+func TestAsyncSpanRace(t *testing.T) {
+	// This tests a regression where asynchronously finishing spans would
+	// modify a flushing root's sampling priority. The test has 100 iterations
+	// because it is not easy to reproduce the race.
+	_, _, stop := startTestTracer()
+	defer stop()
+
+	for i := 0; i < 100; i++ {
+		t.Run("", func(t *testing.T) {
```

Comment:
Is there a convention for naming these subtests when launched in a loop like this?

Reply:
I think this style is perfectly appropriate in this case. Go will automatically name the tests `#00`, `#01`, etc:
```
=== RUN   TestAsyncSpanRace
=== RUN   TestAsyncSpanRace/#00
=== RUN   TestAsyncSpanRace/#01
=== RUN   TestAsyncSpanRace/#02
=== RUN   TestAsyncSpanRace/#03
=== RUN   TestAsyncSpanRace/#04
=== RUN   TestAsyncSpanRace/#05
=== RUN   TestAsyncSpanRace/#06
=== RUN   TestAsyncSpanRace/#07
=== RUN   TestAsyncSpanRace/#08
=== RUN   TestAsyncSpanRace/#09
=== RUN   TestAsyncSpanRace/#10
...
```